project/src/main/webapp/Script.js

```diff
@@ -0,0 +1,79 @@
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//Selects error message within login form
+function setFormMessage(formEmement, type, message) {
+    const messageElement = formEmement.querySelector(".form__message");
+
+    messageElement.textContent = message;
+    messageElement.classList.remove("form__message--success", "form__message--error");
+    messageElement.classList.add(`form__message--${type}`);
+}
+
+//Input Error for individual fields
+function setInputError(inputElement, message) {
+    inputElement.classList.add("form__input--error");
+    inputElement.parentElement.querySelector(".form__input-error-message").textContent = message;
+}
+//Clears Input Error for individual fields
+function clearInputError(inputElement) {
+    inputElement.classList.remove("form__input--error");
+    inputElement.parentElement.querySelector(".form__input-error-message").textContent = "";
+}
+
+//Function links from the create account to the login page and vice versa
+document.addEventListener("DOMContentLoaded", () => {
+    const loginForm = document.querySelector("#login");
+    const createAccountForm = document.querySelector("#createAccount");
+
+    //Hides login form and shows create account form
+    document.querySelector("#linkCreateAccount").addEventListener("click", e => {
+        e.preventDefault();
+        loginForm.classList.add("form--hidden");
+        createAccountForm.classList.remove("form--hidden");
+    });
+
+    // Hides create account form and shows login form
+    document.querySelector("#linkLogin").addEventListener("click", e => {
+        e.preventDefault();
+        loginForm.classList.remove("form--hidden");
+        createAccountForm.classList.add("form--hidden");
+    });
+
+    loginForm.addEventListener("submit", e => {
+        e.preventDefault();
+
+        //Perform Fetch Login..
+
+        setFormMessage(loginForm, "error", "Invalid username/password combination");
+    });
+
+    // error checking for username value
+    document.querySelectorAll(".form__input").forEach(inputElement => {
+        inputElement.addEventListener("blur", e => {
+            if (e.target.id === "signupUsername" && e.target.value.length > 0 && e.target.value.length < 6) {
```

Comment:
Would we be able to remove the condition e.target.value.length > 0 since we just want to make sure the length is not less than 6? Rather than also checking for the length being greater than 0? If I'm misunderstanding something, please let me know.

Reply:
Similarly, for Line 70.

Reply:
Oh neat! I guess you're right about that, I guess it's just a force of habit to make sure there's a minimum value, Thanks